readthedocs/oauth/services/base.py

```diff
@@ -202,11 +202,12 @@ def sync(self):
         all_remote_repositories = remote_repositories + remote_repositories_organizations
         repository_full_names = [r.full_name for r in all_remote_repositories if r is not None]
         (
-            self.user.oauth_repositories
+            self.user.remote_repository_relations
             .exclude(
-                Q(full_name__in=repository_full_names) | Q(project__isnull=False)
+                Q(remoterepository__full_name__in=repository_full_names) |
+                Q(remoterepository__project__isnull=False)
```

Comment:
This restriction is no needed anymore.

We were doing this to avoid disconnecting Projects from RemoteRepository. However, now that we are deleting RemoteRepositoryRelation we can delete the relation without care too much about if the Project is connected to a RemoteRepository or not

Reply:
Okay, Got it! :+1: